docs/integrations/sources/adroll.md

```diff
@@ -0,0 +1,45 @@
+# AdRoll
+AdRoll is an ecommerce marketing platform that gives growing D2C brands the power to connect with customers wherever they are.
+This connector allows you to extract data from various AdRoll APIs such as Advertisables , Ads , Campaigns , Strategies and many more
+Docs : https://developers.adroll.com/apis
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `Authorization` | `string` | Person Access Token (Authorization).  |  |
+| `client_id` | `string` | Client ID. The key ID of your app |  |
+| `client_secret` | `string` | Client secret. The secret of your app |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `oauth_access_token` | `string` | Access token. The current access token. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+| `oauth_token_expiry_date` | `string` | Token expiry date. The date the current access token expires in. This field might be overridden by the connector based on the token refresh endpoint response. |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| Segments | segment_id | DefaultPaginator | ✅ |  ❌  |
+| Campaign | campaign_eid | DefaultPaginator | ✅ |  ❌  |
+| Strategy |  | No pagination | ✅ |  ❌  |
+| Strategies | eid | No pagination | ✅ |  ❌  |
+| Advertisable |  | No pagination | ✅ |  ❌  |
+| Active Pixels for an Advertisable | eid | No pagination | ✅ |  ❌  |
+| Consent Banner Config for an Advertisable |  | No pagination | ✅ |  ❌  |
+| Pixel | eid | No pagination | ✅ |  ❌  |
+| Segments for a Pixel | eid | No pagination | ✅ |  ❌  |
+| Campaigns | eid | DefaultPaginator | ✅ |  ❌  |
+| AdGroups by Advertisables | eid | DefaultPaginator | ✅ |  ❌  |
+| Ads | eid | DefaultPaginator | ✅ |  ❌  |
```

Comment:
Why ads aren't get from https://developers.adroll.com/docs/adroll-ad-api/1/routes/ads/get ?

Reply:
I was facing some error with this endpoint so I used the reporting api 🤔